tests/Tests/Models/ReadonlyProperties/Library.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\Models\ReadonlyProperties;
+
+use Doctrine\ORM\Mapping\Column;
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\GeneratedValue;
+use Doctrine\ORM\Mapping\Id;
+use Doctrine\ORM\Mapping\Table;
+use Doctrine\Tests\Models\ValueObjects\Uuid;
+
+#[Entity]
+#[Table(name: 'library')]
```

Comment:
Does the table name matter? If not, I'd remove that annotation entirely, let's eliminate unnecessary distractions.